pages/index.tsx

```diff
@@ -1,8 +1,76 @@
-import { Button, Code, Title } from "@prisma/lens";
+import { defaultTheme, WebsiteButton } from "@prisma/lens/dist/web";
+import * as reactCanvas from '@rive-app/react-canvas';
 import Head from "next/head";
 import Image from "next/image";
-import { useState } from "react";
+import { useEffect, useState } from "react";
+import { useInView } from 'react-intersection-observer';
 
+import DatabaseInfo from "../components/DatabaseInfo";
+import styles from "../styles/index.module.scss"
+
+const Animation = ({ name, className, fit, autoplay }: any) => {
+  const [autoplayState, setAutoplay] = useState<boolean>(autoplay)
+
+  const [reference, isInView] = useInView({
+    triggerOnce: true,
+    // rootMargin: "200px"
+    threshold: 0.2,
+  });
+
+  const { rive, RiveComponent } = reactCanvas.useRive({
+    src: `/animations/${name}.riv`,
+    autoplay: autoplayState,
+    layout: new reactCanvas.Layout({
+      fit: fit,
+      alignment: reactCanvas.Alignment.Center,
+    }),
+  });
+
+  useEffect(() => {
+    if (isInView) rive?.play();
+  }, [isInView])
+
+  useEffect(() => {
+    if (isInView) {
+      setAutoplay(true)
+      rive?.play();
+    }
+  }, [])
+
+  return <div ref={reference} className={className}><RiveComponent /></div>
+}
+
+const pageInfo = [
+  {
+    title: "Application",
```

Comment:
the style of these elements doesn't seem to be the exact one from the designs yet.

current:
<img width="803" alt="image" src="https://github.com/prisma/accelerate-speed-test/assets/4058327/2304245f-940a-4812-9e19-5ea08c4bdfab">


[design](https://www.figma.com/file/kF1TWqqWUdHg7TUZr97jmt/ACCELERATE?type=design&node-id=335%3A30724&mode=design&t=zMiW6gV8g70K5LuH-1):
<img width="579" alt="image" src="https://github.com/prisma/accelerate-speed-test/assets/4058327/3876c486-6842-4091-875a-12046501e50a">

Reply:
@carlagn

Reply:
@nikolasburk We don't have the lighter font file available so Alejandro updated the designs to use the semibold, which I've updated on my last commit.